CMakeLists.txt

```diff
@@ -9,6 +9,25 @@ set(CMAKE_VS_INCLUDE_INSTALL_TO_DEFAULT_BUILD 1)
 
 set(PROJECT "HDILib")
 PROJECT(${PROJECT})
+
+# Disallow in-source builds. 
+# Build in sub dir e.g. source/build* is still allowed!
+if("${PROJECT_SOURCE_DIR}" STREQUAL "${PROJECT_BINARY_DIR}" AND NOT $ENV{CI})
```

Comment:
Just for my understanding, does this `AND NOT $ENV{CI}` mean that the CI actually does in-source-build?